packages/theme-chalk/src/segmented.scss

```diff
@@ -28,13 +28,23 @@ $segmented-font-size: map.merge(
 $segmented-item-padding: () !default;
 $segmented-item-padding: map.merge(
   (
-    'large': 0 11px,
-    'default': 0 11px,
-    'small': 0 7px,
+    'large': 11px 11px,
```

Comment:
The value here should probably not be modified, or only in the vertical direction, to avoid affecting the original default style.

Reply:
done.